amundsen_application/static/js/components/SearchPage/SearchBar/index.tsx

```diff
@@ -11,11 +11,16 @@ import {
   SYNTAX_ERROR_PREFIX,
   SYNTAX_ERROR_SPACING_SUFFIX,
 } from './constants';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { GlobalState } from 'ducks/rootReducer';
+import { connect } from 'react-redux';
+
+export interface StateFromProps {
+  searchTerm: string;
+}
 
 export interface SearchBarProps {
```

Comment:
You can now call this `OwnProps` and declare:
`export type SearchBarProps = StateFromProps & OwnProps & RouteComponentProps<any>;`
and then be able to keep:
 `export class SearchBar extends React.Component<SearchBarProps, SearchBarState>`